module/Finna/src/Finna/RecordDriver/SolrLido.php

```diff
@@ -576,7 +576,9 @@ protected function getRepresentations(string $language): array
                     || ($displayAsLink = in_array($type, $this->displayExternalLinks))
                 ) {
                     $documentDesc = $description;
-                    if ($displayAsLink ??= false && !$documentDesc) {
+                    $displayAsLink ??= false;
```

Comment:
Laittaisin mieluummin niin, että `$displayAsLink = in_array($type, $this->displayExternalLinks)` tehdään ennen if-lausetta. Muutenhan tuo voisi periaatteessa mennä falseksi silloinkin kuin ei pitäisi.